src/main/resources/db/migration/0250/V277__NewParticipantProjectSpeciesFields.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE accelerator.participant_project_species ADD COLUMN internal_comment TEXT;
+
+CREATE TABLE native_non_native (
```

Comment:
The name here seems a bit too specific for this to be an enum. If we add a third choice at some point, then `native_non_native` might become a misleading name for it. If we never add a third choice, the native status could be a boolean column on `participant_project_species` instead of an enum.

If we keep it as an enum, something like `species_native_types` or `species_native_categories` would still be accurate if there are more choices later, and (more subjectively, I admit) names like those also make it clearer at a glance that the table is an enum table that relates to species.

Reply:
Good call. I asked about this in the PRD but haven't gotten an answer about a more general name for it. What you suggest seems like a good alternative.